apisix/cli/ngx_tpl.lua

```diff
@@ -617,6 +642,17 @@ http {
             proxy_pass $upstream_mirror_host$request_uri;
         }
         {% end %}
+
+        location = /50x.html {
```

Comment:
Would it be better to use `named location`?

Reply:
Not really good to do that.Because we will use "50x.html" to search in the "html" directory.

Reply:
So the log phase might get a rewritten URL instead of the original one? This is not good news...

Reply:
Oh, I found a new way. We can use "try_files" to get the html.

Reply:
Fixed. Please review it again.